autoblocks/_impl/testing/run.py

```diff
@@ -2,63 +2,41 @@
 import contextvars
 import dataclasses
 import inspect
-import threading
 import traceback
 from typing import Any
 from typing import Callable
-from typing import Coroutine
 from typing import List
 from typing import Optional
 
 import httpx
 import orjson
 
+from autoblocks._impl import global_state
 from autoblocks._impl.testing.models import BaseEvaluator
 from autoblocks._impl.testing.models import BaseTestCase
 from autoblocks._impl.testing.models import Evaluation
 from autoblocks._impl.util import AutoblocksEnvVar
+from autoblocks._impl.util import gather_with_max_concurrency
 
-# Globals
+# httpx client
```

Comment:
probably still worth leaving the word "global" in this comment, e.g. "Global httpx client"

Reply:
Added Global